crates/astria-core/src/sequencer/v1alpha1/celestia.rs

```diff
@@ -32,7 +32,10 @@ impl CelestiaBlobBundle {
             rollup_ids_proof,
         } = block.into_unchecked();
 
-        let tree = super::derive_merkle_tree_from_rollup_txs(&rollup_transactions);
+        let rollup_base_transactions = rollup_transactions
+            .iter()
+            .map(|(rollup_id, txs)| (rollup_id, txs.transactions()));
+        let tree = super::derive_merkle_tree_from_rollup_txs(rollup_base_transactions);
```

Comment:
What happens if this tree is constructed from filtered block? That is, the sequencer block yielded by the transformation `filtered_block(SequencerBlock, [RollupId]) -> SequencerBlock`? Isn't this tree wrong?